src/main/java/edu/greenblitz/tobyDetermined/subsystems/intake/IntakeRoller.java

```diff
@@ -0,0 +1,45 @@
+package edu.greenblitz.tobyDetermined.subsystems.intake;
```

Comment:
No get functions?
No states?
Can you roll in an object while extender is closed?